storybook/stories/API/chart/BarChart.stories.tsx

```diff
@@ -1,9 +1,188 @@
 import React from 'react';
+import { Args } from '@storybook/react';
 import { Bar, BarChart, ResponsiveContainer, XAxis } from '../../../../src';
 import { pageData } from '../../data';
 
+const GeneralProps: Args = {
+  layout: {
+    description: 'The layout of bars in the chart.',
+    table: {
+      type: {
+        summary: "'horizontal' | 'vertical'",
+        defaultValue: "'horizontal'",
+      },
+      category: 'General',
+    },
+  },
+  syncId: {
+    description: `If any two categorical charts(LineChart, AreaChart, BarChart, ComposedChart) have the same syncId, 
```

Comment:
nit nit: can we add a space between charts and the parenthesis? "categorical charts (LineChart..."